src/pages/index.js

```diff
@@ -274,20 +273,20 @@ avatarImageFormValidator.enableValidation();
 
   cardSection.renderItems();
 });
- */
+
 /* Fetch and displaying User name, job bio, and profile picture */
-/* api.getUserInformation().then((userData) => {
+api.getUserInformation().then((userData) => {
   userId = userData._id;
   profileTitle.textContent = userData.name;
   profileJob.textContent = userData.about;
   userProfileAvatar.src = userData.avatar;
-}); */
+});
 
 /* -------------------------------------------------------------------------- */
 /*                      Avatar picture change popup form                      */
 /* -------------------------------------------------------------------------- */
 
-/* function handleAvatarImageServerSubmit() {
+function handleAvatarImageServerSubmit() {
   submitButtonChangeAvatar.textContent = "Saving...";
   api
     .updateProfilePicture({ avatar: avatarImageInput.value })
```

Comment:
* Only method `_getInputValues` collects all the data of the inputs and passes it to the call of `submitForm`. So you need to use the collected data rather than the inputs.